eng/pipelines/templates/jobs/archetype-sdk-client.yml

```
@@ -219,13 +161,41 @@ jobs:
         - pwsh: npm install -g moxygen
           displayName: Install Moxygen to generate markdown for docs.microsoft.com
 
+        - pwsh: |
+            Write-Host "Using apiview parser version ${{ variables.apiviewParserVersion }}"
```

I don't think it will render properly with the `${{` compile time syntax.

```suggestion
            Write-Host "Using apiview parser version $(apiviewParserVersion)"
```